cmd/osv-scanner/scan/image/command_test.go

```diff
@@ -283,6 +319,36 @@ func TestCommand_OCIImage_JSONFormat(t *testing.T) {
 				testcmd.AnyDiffID,
 			},
 		},
+		{
+			Name: "scanning_insecure_alpine_image_with_specific_detector_enabled",
+			Args: []string{
+				"", "image", "--format=json",
+				"--experimental-detectors", "weakcredentials/etcshadow",
+				"--archive", "../../../../internal/image/fixtures/test-alpine-etcshadow.tar",
+			},
+			Exit: 1,
+			ReplaceRules: []testcmd.JSONReplaceRule{
+				testcmd.GroupsAsArrayLen,
+				testcmd.OnlyIDVulnsRule,
+				testcmd.OnlyFirstBaseImage,
+				testcmd.AnyDiffID,
+			},
+		},
+		{
+			Name: "scanning_insecure_alpine_image_with_detector_preset",
+			Args: []string{
+				"", "image", "--format=json",
+				"--experimental-detectors", "weakcreds",
```

Comment:
I'm surprised this is running at all, since in ValidatePluginRequirements in scalibr it should fail if we are running some of these extractors on containers.